### ISSUE TITLE
Any results?

### ISSUE THREAD

**Issue by smolendawid** [HawkAaron/E2E-ASR#2]
Are there any results on any standard dataset?

**Comment by HawkAaron** [comment]
Since I'm working on [RNN Transducer loss implementation](https://github.com/HawkAaron/warp-transducer), after finish that, I'll create a repo release the TIMIT training & decoding code on TensorFlow, MXNet and PyTorch.

**Comment by holianh** [comment]
in this repo, you use local/ folder, contain some code, can you provide me full your code?
I am glad waiting for your full code here or email: tuananhktmt@gmail.com
thank you so much!

**Comment by HawkAaron** [comment]
@holianh The local/ folder is from kaldi, so you can find it in [kaldi timit recipe](https://github.com/kaldi-asr/kaldi/tree/master/egs/timit/s5).

**Comment by guangsenw** [comment]
is the rnnt loss finished? could you please release some numbers on the TIMIT set also the parameter settings for training the rnnt model?

**Comment by HawkAaron** [comment]
The results for CTC and RNN-T are released.
